src/nnet3bin/nnet3-copy.cc

```diff
@@ -91,8 +94,11 @@ int main(int argc, char *argv[]) {
     if (scale != 1.0)
       ScaleNnet(scale, &nnet);
     
-    if (dropout > 0)
-      SetDropoutProportion(dropout, &nnet);
+    if (dropout >= 0)
+      KALDI_ERR << "--dropout option is deprecated. "
```

Comment:
[this for Vimal, but]... "deprecated" means still supported but not recommended.  Since this will crash, deprecated is not the right term.  I would suggest to just remove the set-dropout-proportion option and related code from the program.  If no scripts are using it, this is OK.